app/views/personaldetails/UpdateAddressView.scala.html

```diff
@@ -14,14 +14,13 @@
  * limitations under the License.
  *@
 
-@import uk.gov.hmrc.play.views.html._
 @import uk.gov.hmrc.play.views.html.helpers._
 @import tags._
 @import config.ConfigDecorator
 @import controllers.auth.requests.UserRequest
 @import util.LocalPartialRetriever
 @import uk.gov.hmrc.renderer.TemplateRenderer
-
+@this(form: FormWithCSRF, input: Input)
```

Comment:
Is input used here?

Reply:
Yes, there are 6 uses in this file, I've done this because it triggers a warning to DI components like this as the static references are deprecated as of Play 2.6